docs/developer_portal/extensions/registry.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Community Extensions
+sidebar_position: 9
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Community Extensions
+
+This page serves as a registry of community-created Superset extensions. These extensions are developed and maintained by community members and are not officially supported or vetted by the Apache Superset project. **Before installing any community extension, administrators are responsible for evaluating the extension's source code for security vulnerabilities, performance impact, UI/UX quality, and compatibility with their Superset deployment.**
+
+## Extensions
+
+| Name | Description | Author | Preview |
+|------|-------------|--------|---------|
+| [Query Insights](https://github.com/example/superset-query-insights) | A SQL Lab extension that displays metadata and data quality information about tables referenced in your queries. Shows table descriptions, data quality scores, latest partition dates, data retention policies, team ownership, and example queries. | Michael S. Molina | <a href="/img/extensions/example.png" target="_blank"><img src="/img/extensions/example.png" alt="Query Insights" width="120" /></a> |
```

Comment:
Should we maintain a real branch for the example extension? Having a link to a 404 page will likely not be a great experience.

Reply:
I developed two extensions that I'm planning to host on my personal repository:
- An extension that exports SQL Lab results to Google Sheets
- An extension that display core API events

My plan is to replace the Query Insights example with those ([task](https://github.com/orgs/apache/projects/544?pane=issue&itemId=142447039)). I only added the Query Insights so we can see how this page will look like in the future. I think we can merge this PR and open a follow-up.